app/controllers/hub/sites_controller.rb

```diff
@@ -27,9 +27,14 @@ def edit
     end
 
     def update
-      render :edit unless @vita_partner.update(vita_partner_params)
+      if @vita_partner.update(vita_partner_params)
```

Comment:
There may be merge conflicts between this and #777 . Let's merge your PR #781 first, and then I can fix any merge conflicts that arise for me.

Reply:
If you want to merge #777 while you're merging this, and then go update the Pivotal Tracker story to be Delivered, I would enjoy that, but it's really not essential.